### DISCUSSION TITLE
schedule for next release?

### DISCUSSION THREAD

**Discussion by harmsm** [sandialabs/toyplot#206] [General]
Is there a plan for the next release date? The current 1.0.1 release is incompatible with the latest version of numpy (1.22), meaning conda recipes have to pin numpy to use toytree. This is already leading to dependency conflicts for me. The current source code is compatible with 1.22, so this should be resolved on the next release, whenever that is. Any plan for timing on that? Thanks for the great library!

**Comment by tshead2** [comment]
Thanks for staying on top of this, I shouldn't have let it go so long.  Toyplot 1.0.2 is released with your changes ... thanks again!
Tim

**Comment by harmsm** [comment]
Hi Tim,

Thanks! This stuff is the most annoying part of having an open source package… 

Best,

Mike
